src/nodemanagerbase.cpp

```diff
@@ -19,54 +19,50 @@
  *  along with Quasar.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <algorithm>
+#include <stdexcept>
 
 #include <nodemanagerbase.h>
-#include <boost/foreach.hpp>
-#include <iostream>
 #include <opcua_basedatavariabletype.h>
-#include <stdexcept>
 #include <uadatavariablecache.h>
 
 static UaLocalizedText emptyDescription( "en_US", "" );
 
-NodeManagerBase::NodeManagerBase( const char* uri, bool sth, int hashtablesize ):
-    m_server(0),
-    m_nameSpaceUri(uri)
+NodeManagerBase::NodeManagerBase(
+        const char* sNamespaceUri,
+        bool        firesEvents,
+        OpcUa_Int32 nHashTableSize):
+    m_server(nullptr),
+    m_nameSpaceUri(sNamespaceUri)
 {
 
 }
 
 NodeManagerBase::~NodeManagerBase()
 {
-
-
-    std::cout << __FUNCTION__ << std::endl;
```

Comment:
maybe worth a 
`LOG(TRC) << __FUNCTION__`
in new code

Reply:
I think this was really some unnecessary log output from the early days, I don't think it should stay.